resources/assets/js/components/Twitter.vue

```diff
@@ -110,9 +110,9 @@
                     return;
                 }
 
-                this.onDisplay.unshift(this.waitingLine.shift());
+                this.tweets.unshift(this.waitingLine.shift());
 
-                this.onDisplay = this.onDisplay.slice(0,10);
+                this.tweets = this.tweets.slice(0,10);
```

Comment:
We could potentially be hiding a lot of tweets, maybe up this to 20? Could be that `initial-tweets` would need an increase too.